Sources/Unbox.swift

```diff
@@ -61,6 +68,13 @@ public func Unbox<T: Unboxable>(dictionary: UnboxableDictionary, at key: String,
     return container.models
 }
 
+/// Unbox an array JSON dictionary into an array of model `T` beginning at a provided key, optionally using a contextual object and/or invalid elements. Throws `UnboxError`.
+public func Unbox<T: UnboxableWithContext>(dictionary: UnboxableDictionary, at key: String, isKeyPath: Bool = true, context: T.ContextType? = nil) throws -> [T] {
```

Comment:
The documentations says that this function supports allowing invalid elements, but it doesn't 😅 Also make `context` non-optional.